### DISCUSSION TITLE
Is the h3 extension still supported?

### DISCUSSION THREAD

**Discussion by marklit** [duckdb/duckdb#7860] [General]
I've been working off the main branch for a few months now and usually the H3 extension is ready pretty soon after a release. Is it still supported now?

```sql
D INSTALL h3;
Error: HTTP Error: Failed to download extension "h3" at URL "http://extensions.duckdb.org/57912b60a6/linux_amd64/h3.duckdb_extension.gz"

Candidate extensions: "s3", "http", "https", "httpfs", "md"
D 
```

**Comment by Mause** [comment]
There is no official h3 extension - perhaps you're thinking of https://github.com/isaacbrodsky/h3-duckdb?

**Comment by marklit** [comment]
I must be confused. I thought I was pulling it from your repo but the H3 extension is probably hidden inside some bash script I have that builds DuckDB and the Spatial extension for me.

**Reply by Mause** [reply]
It's possible you had a custom extension repository set and were downloading it that way yes
